libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1034,24 +1034,32 @@ void AP_GPS::Write_DataFlash_Log_Startup_messages()
 
 /*
   return the expected lag (in seconds) in the position and velocity readings from the gps
```

Comment:
This should now say get the expected lag, rather then return. The double return comments are confusing on a first pass.